src/hotspot/os/windows/os_windows.cpp

```diff
@@ -1236,7 +1236,18 @@ void os::die() {
 const char* os::dll_file_extension() { return ".dll"; }
 
 void  os::dll_unload(void *lib) {
-  ::FreeLibrary((HMODULE)lib);
+  char name[MAX_PATH];
+  if (::GetModuleFileName((HMODULE)lib, name, sizeof(name)) == 0) {
+    name[0] = '\0';
```

Comment:
Should we set `<not available>` to `name` like a change for linux code?

Reply:
Yes, it is most likely better to have same output in failure case on Windows and other platforms.